README.md

```diff
@@ -47,9 +47,10 @@ ddbt version 0.6.3
 - `--models model_filter` _or_ `-m model_filter`: Instead of running for every model in your project, DDBT will only execute against the requested models. See filters below for what is accepted for `my_model`
 - `--threads=n`: force DDBT to run with `n`  threads instead of what is defined in your `dbt_project.yml`
 - `--target=x` _or_ `-t x`: force DDBT to run against the `x` output defined in your `profile.yml` instead of the default defined in that file.
-- `--upstream=y` _or_ `-u y`: For any references to models outside the explicit models specified by run or test, the upstream target used to read that data will be swapped to `y` instead of the output target of `x`  
-- `--fail-on-not-found=false` _or_ `-f=false`: By default, ddbt will fail if a the specified models don't exist, passing in this argument as false will warn instead of failing  
-- `--enable-schema-based-tests` _or_ `-s=true`: Schema-based tests are disabled by default for now, but as a way to enable them pass this argument as true 
+- `--upstream=y` _or_ `-u y`: For any references to models outside the explicit models specified by run or test, the upstream target used to read that data will be swapped to `y` instead of the output target of `x`
+- `--fail-on-not-found=false` _or_ `-f=false`: By default, ddbt will fail if a the specified models don't exist, passing in this argument as false will warn instead of failing
+- `--enable-schema-based-tests` _or_ `-s=true`: Schema-based tests are disabled by default for now, but as a way to enable them pass this argument as true
+- `--custom-config-path=myCustomPathWithTrailingSlash/` _or_ `-c=myCustomPathWithTrailingSlash/`: Allows a custom path to be used for the dbt config and project files. This is useful if you want to use a different location than the default one. For example if you're mid-way through migrating commands from an old dbt version to a new version and using two different versions of config at the same time.
```

Comment:
I don't think the description matches the implementation? It looks like the implementation only sets the path for the `dbt_project.yml` file, whereas this implies it will set the path for multiple files?

Reply:
Good catch - a flaw with the Docs first approach! Pushed a change.